### ISSUE TITLE
Help for generating part masks of my own dataset

### ISSUE THREAD

**Issue by ZhangYiyangKevin** [zhangxinyu-xyz/PGAN-VehicleRe-ID#2]
Dear author, 
    I get confused about generating masks of my own dataset. Can you give me some materials such as models and pretrained weight? I failed to search any infos about the pre-trained SSD detector trained on vehicle attributes refering to [Structural Analysis of Attributes for Vehicle Re-Identification and Retrieval]. 
    Look forward to your reply！

**Comment by ilsunshine** [comment]
Mr.Zhang:
    I have the same problem as you. Have you solved the problem of this SSD detector. If you solve the problem, would you please tell me how you solved it
   Look forward for your reply!
